### ISSUE TITLE
Blockchain Fully Synched, but my balance is ZERO

### ISSUE THREAD

**Issue by donaghq** [ethereum/mist#3848]
* [x] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

```
Version: `0.10.0`
OS & Version: windows
Node version: `geth 1.8.2`
Number of blocks synchronized: complete 
```

**Comment by donaghq** [comment]
checking

**Comment by donaghq** [comment]
![geth](https://user-images.githubusercontent.com/38651766/39142340-4d1f9a46-4722-11e8-949e-37c64ef5cded.png)

**Comment by donaghq** [comment]
no such thing as support?

**Comment by ryanio** [comment]
are you running geth separately from mist? if you close geth and open mist, it should run its own instance which should be connected. what block number does the sidebar say you're on?

sorry for the trouble!

**Comment by donaghq** [comment]
i think I will abondon this and use MEW with JSON file - not the best security but this will not work. I have tried running geth seperately. MIST gets stuck at downloading....says its got 65 to  download. never syncs and never display my balance

**Comment by ryanio** [comment]
hey, sorry you can't get to 100%. we are working on a new release (should be out in a few weeks) that should solve all these syncing problems by allowing you to initially connect to a remote(infura) node while your local node syncs in the background. keep an eye out! meanwhile MEW works well, we would recommend using it with [MetaMask](https://metamask.io/)

**Comment by donaghq** [comment]
thanks!

**Comment by ryanio** [comment]
Hey, we have now released a new version (`0.11.0`) that has new syncing features - please give it a try! If you have any new problems that you can reproduce, please create a new issue with the details on how 
we can help. Thanks!

**Comment by donaghq** [comment]
Thank you! Would be nice if the Blockchain was distributed. The chain is
100G. Hard of manage if not local to you. Your working machine. Just need a
dedicated drive I guess. Been using Meta mask which is seems good and
hassle free. Your thoughts?

DQ
